sphere-models/src/main/java/io/sphere/sdk/carts/Cart.java

```diff
@@ -79,4 +93,8 @@ public String toString() {
 
     @Override
     List<DiscountCodeInfo> getDiscountCodes();
+
+    @Nullable
+    @Override
+    CustomFields getCustom();
```

Comment:
I know this name is coming from the platform, but do you know why is it not `CustomFields` or similar?